astro/src/content/docs/get-started/download-and-install/docker.mdx

```diff
@@ -223,14 +128,22 @@ When running in development runtime mode, silent mode was enabled based upon the
 The Docker Compose file references the `latest` tag, but that tag is not dynamic. It is only the latest at a point in time. To get the most recently released image, you have a couple of options.
 
 * Edit the Docker Compose file and specify a specific version. This is a good idea for a production deployment.
-* Remove the old image first: `docker rmi <old image id>`. This command may prompt you to remove containers using that image. Since all state is stored in the database, you can safely remove the containers. Then, when you restart Docker Compose, the latest image will be retrieved.
-* Pull the latest image with this command: `docker pull fusionauth/fusionauth-app:latest`.
+* Remove the old image first: `docker rmi <old image id>`. This command may prompt you to remove containers using that image. Since all state is stored in the database, you can safely remove the containers. Then, when you recreate Docker Compose, the latest image will be retrieved.
```

Comment:
You aren't recreating docker compose, you are restarting docker compose which recreates the network, pulls images, etc. What am I missing?

Reply:
I completely redone the section, and describing any of it as restart doesn't make sense in general, unless we use docker compose stop/start/restart commands.